common/lib/xmodule/xmodule/modulestore/xml_importer.py

```diff
@@ -503,7 +504,6 @@ def depth_first(subtree):
             except Exception:
                 msg = f'Course import {self.target_id}: failed to import module location {leftover}'
                 log.error(msg)
-                set_custom_attribute('course_import_failure', f"Module Load failure: {msg}")
```

Comment:
The code is reraising the exception which will be caught and tracked at **CaptureRootBroadExcept**.